src/tests/src/solver/infeasible-problem-analysis/test-unfeasible-problem-analyzer.cpp

```diff
@@ -171,9 +171,9 @@ std::unique_ptr<MPSolver> createUnfeasibleProblem(const std::string& constraintN
 }
 
 static const std::string validConstraintNames[] = {
-  "BC::hourly::hour<36>",
-  "BC::daily::day<67>",
-  "BC::weekly::week<12>",
+  "BC-name-1::hourly::hour<36>",
```

Comment:
Here I changed a bit the binding constraints' names.
Originally, these names started with "BC::", and it made think that BC names in **mps** files necessarily start with this prefix. This not the case, this prefix is the name of the BC.